src/Core/Style.js

```diff
@@ -755,14 +777,83 @@ class Style {
         return this;
     }
 
+    /**
+     * Applies the style.fill to a polygon of the texture canvas.
+     * @param {CanvasRenderingContext2D} txtrCtx The Context 2D of the texture canvas.
+     * @param {Path2D} polygon The current texture canvas polygon.
+     * @param {Number} invCtxScale The ratio to scale line width and radius circle.
+     * @param {Boolean} canBeFilled - true if feature.type == FEATURE_TYPES.POLYGON.
+     */
+    applyToCanvasPolygon(txtrCtx, polygon, invCtxScale, canBeFilled) {
+        // draw line or edge of polygon
+        if (this.stroke) {
+            // TO DO add posibility of using a pattern
+            Style.prototype._drawPolygon.call(this, txtrCtx, invCtxScale);
```

Comment:
I think `_applyStrokeToPolygon` would be a better name to avoid confusion with `Feature2Texture._drawPolygon`

Reply:
Indeed, I was thinking also about a name but didn't find a better one. I used your suggestion.